jerry-ext/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required (VERSION 2.8.12)
+set(JERRY_EXT_NAME jerry-ext)
+project (${JERRY_EXT_NAME} C)
+
+# Include directories
+set(INCLUDE_EXT "${CMAKE_CURRENT_SOURCE_DIR}/include")
+
+# Source directories
+file(GLOB SOURCE_EXT args/*.c)
+
+add_library(${JERRY_EXT_NAME} STATIC ${SOURCE_EXT})
+
+target_include_directories(${JERRY_EXT_NAME} PUBLIC ${INCLUDE_EXT})
+target_include_directories(${JERRY_EXT_NAME} PRIVATE "${CMAKE_SOURCE_DIR}/jerry-core")
```

Comment:
I removed the `SYSTEM`, so the comments from @akosthekiss  and my reply are hidden. **But the problem is not solved yet**, so I paste them here again.

@akosthekiss : Is this needed? As jerry-ext is dependent on jerry-core, all interface include dirs of jerry-core will be added to the include path when compiling jerry-ext. From the docs:
```
Targets [...] publish the [INTERFACE_INCLUDE_DIRECTORIES] required to compile against the headers for the target.
```

@jiangzidong : It should behave like you said. But when I remove this line, 
```
jerry-ext/include/jerryscript-ext/args.h:23:25: fatal error: jerryscript.h: No such file or directory
 #include "jerryscript.h"
```

But when I remove the similar cmd in jerry-core/CMakeLists line 257, 262
```
target_include_directories(${JERRY_CORE_NAME} SYSTEM PRIVATE "${CMAKE_SOURCE_DIR}/jerry-libm/include")
```
and 
```
target_include_directories(${JERRY_CORE_NAME} SYSTEM PRIVATE "${CMAKE_SOURCE_DIR}/jerry-libc/include
```
The jerry-core works well.

I don't know why `jerry-ext` needs the explicit header set, while `jerry-core` doesnt.

---

We can continue in this thread